### PR TITLE
Set up CI using Github actions

### LINKED ISSUE #13: CI. Handle Continuous integration
Ensure continuous integration.

Implement TravisCI

Alternatively, CircleCI.

### PR DESCRIPTION
**Pull request by vector-ok**
## Description
CI set up using Github Actions. This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node for the develop branch only.
It removes the need for a third party framework like the TravisCI originally picked.
Fixes #13

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npm run build --if-present
+    - run: npm test
```
Comment (ekrresa) [inline .github/workflows/nodejs.yml:28]: You may need to change this from `npm` to `yarn` since already there's a `yarn.lock` file in the repo

**Review by ekrresa — APPROVED** [review]
Looks good. Just switch from `npm` to `yarn`, unless we decided to use npm

File: .github/workflows/nodejs.yml (hunk 1 above)
Comment (vector-ok) [reply .github/workflows/nodejs.yml:28]: good observation

**Comment by vector-ok** [comment]
we are using npm.

**Pull request merged**